apps/payments/next/app/page.tsx

```diff
@@ -11,52 +12,55 @@ export default function Index() {
   // and instead redirects to the Subscription Management page.
   // This page will be fixed before launch by FXA-8304
   return (
-    <div>
-      <h1 className="text-xxl text-center m-4">Welcome</h1>
-      <div className="flex-col">
-        <h2 className="text-xl">With auth</h2>
-        <div className="flex gap-8">
-          <div className="flex flex-col gap-2 p-4 items-center">
-            <h2>VPN - Monthly</h2>
-            <Link
-              className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
-              href="/en/vpn/monthly/landing"
-            >
-              Redirect
-            </Link>
+    <>
+      <Header />
```

Comment:
Header is not necessary here.

Reply:
Just to elaborate a little. This page is just a temporary page to help with development, and will be removed in the near future.